mission-vision.md

```diff
@@ -0,0 +1,30 @@
+# OpenFeature Mission and Vision
+
+## Mission
+
+To improve the software development lifecycle, no matter the size of the project, by standardizing feature flagging for developers.
+
+## Vision
+
+A world where feature flagging is a core principle of the software development lifecycle, enabling developers to test and release high quality software with a high degree of safety and confidence.
+
+OpenFeature can achieve this vision by:
+
+### Vendor neutrality
+
+OpenFeature aims to be the primary mechanism an [application author][glossary-app-auth] interacts with when using feature flags in code by standardizing the developer facing interfaces in a vendor neutral way. This provides flexibility for the [application integrators][glossary-app-int] to choose the tooling that best fits their current requirements, while avoiding code-level lock-in.
+
+### Extensibility
+
+OpenFeature should be extensible while still being easy to work with. It should be quick to get started with, while providing [application authors][glossary-app-auth] the flexibility to customize the functionality based on their business requirements and workflows.
+
+### Community
+
+OpenFeature is a collective effort that benefits from years of experience across multiple verticals, including many of the top feature flag vendors and practitioners. The common understanding of feature flags and their use in software, developed by this collaboration, will enable feature flags to become a core principle of the software development lifecycle.
```

Comment:
A bit of a bigger change to the last sentence, that might be clearer and read better.

```suggestion
OpenFeature is a collective effort that benefits from years of experience across multiple verticals, including many of the top feature flag vendors and practitioners. The patterns, idioms, and techniques developed by this collaboration will enable feature flagging to become a core principle of the software development lifecycle.
```